<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>快捷键菜单</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="q_Build">
         <property name="text">
          <string>新建</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="q_Open">
         <property name="text">
          <string>打开</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="q_Save">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="q_SaveAs">
         <property name="text">
          <string>另存为</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="q_Undo">
         <property name="text">
          <string>撤销</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="q_Redo">
         <property name="text">
          <string>取消撤销</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="q_Type">
         <property name="text">
          <string>字体</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="q_Color">
         <property name="text">
          <string>颜色</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="Text"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>518</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="wenjian">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actBuild"/>
    <addaction name="actOpen"/>
    <addaction name="actSave"/>
    <addaction name="actSaveAs"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="geshi">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="actType"/>
    <addaction name="actColor"/>
   </widget>
   <widget class="QMenu" name="bianji">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actUndo"/>
    <addaction name="actRedo"/>
    <addaction name="actCut"/>
    <addaction name="actCopy"/>
    <addaction name="actPaste"/>
    <addaction name="actAllSeclect"/>
    <addaction name="actTime"/>
   </widget>
   <addaction name="wenjian"/>
   <addaction name="bianji"/>
   <addaction name="geshi"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actBuild">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actType">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="actUndo">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="actCut">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>剪贴</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actAllSeclect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
  </action>
  <action name="actTime">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actRedo">
   <property name="text">
    <string>取消撤销</string>
   </property>
  </action>
  <action name="actColor">
   <property name="text">
    <string>颜色</string>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
